.travis.yml

```diff
@@ -11,5 +10,12 @@ gemfile:
   - gemfiles/rails4_0.gemfile
   - gemfiles/rails4_1.gemfile
   - gemfiles/rails4_2.gemfile
+  - gemfiles/rails5_0.gemfile
```

Comment:
We should drop support for 3.2 in the next major version (which can be just after this).